lib/ts/recipe/session/recipe.ts

```diff
@@ -227,4 +239,20 @@ export default class SessionRecipe extends RecipeModule {
             userContext: {},
         });
     };
+
+    addDefaultClaimBuilder = (builder: SessionClaimBuilder<any>) => {
+        this.defaultClaimsBuilders.push(builder);
+    };
+
+    getDefaultClaimBuilders = (): SessionClaimBuilder<any>[] => {
+        return this.defaultClaimsBuilders;
+    };
+
+    addDefaultClaimValidator = (builder: SessionClaimValidator) => {
+        this.defaultClaimValidators.push(builder);
+    };
+
+    getGlobalClaimValidators = (): SessionClaimValidator[] => {
+        return this.defaultClaimValidators;
+    };
```

Comment:
This was a mistake, I renamed (search&replace) something too eagerly, they definitely should match.
IMHO: 
- the internally used (= on the recipe) should be called default (`defaultGlobal`?), 
- which is run through the overrideable `getGlobalX` to get the actual global list.
I prefer this because it separates the two concepts nicely.

Reply:
Okay. We can keep them different names, but I don't like default prefix. it's very confusing.

Also, why do we have to call these differently in the first place (more terms is bad)?

Reply:
More terms can be bad, but I think we should differentiate between functions only called internally and user-facing (and overrideable) functions. It's a bit easier for me to see what to expect. Let's discuss naming them on a call.